app/controllers/items_controller.rb

```diff
@@ -15,6 +10,12 @@ def create
     end
   end
 
+  def index
+    @category = Category.new
+    @item = Item.new
+    @items = Item.valid_items
+  end
+
```

Comment:
Would like to merge this branch and then create a new one with presenter logic, so as not to make this pull request massive. Ok with everyone @BrandonMathis @joeyjoejoejr?